### ISSUE TITLE
Cannot build 1.6.1 from source

### ISSUE THREAD

**Issue by FollieHiyuki** [miguelmota/cointop#92]
Mismatched version with vendor module:
```
go: inconsistent vendoring in /builddir/cointop-1.6.1:
	github.com/miguelmota/go-coinmarketcap@v0.1.7: is explicitly required in go.mod, but vendor/modules.txt indicates github.com/miguelmota/go-coinmarketcap@v0.1.6
```

Build with `go get github.com/miguelmota/cointop` or by manually changing the specified module version:
```
# github.com/miguelmota/cointop/pkg/api/impl/coinmarketcap
.local/share/go/src/github.com/miguelmota/cointop/pkg/api/impl/coinmarketcap/coinmarketcap.go:161:3: unknown field 'Slug' in struct literal of type "github.com/miguelmota/cointop/vendor/github.com/miguelmota/go-coinmarketcap/pro/v1".InfoOptions
```

**Comment by miguelmota** [comment]
@FollieHiyuki Thanks for the report! Re-ran `go mod vendor`, can you try again?

**Comment by FollieHiyuki** [comment]
Thank you. It builds correctly now.
Can you make a release just for the sake of packaging? That would be nice.

**Comment by miguelmota** [comment]
@FollieHiyuki Yes, just published [v1.6.2](https://github.com/miguelmota/cointop/releases/tag/v1.6.2)

**Comment by FollieHiyuki** [comment]
Thanks. It works on my system.
@miguelmota I think we can close this issue now.
